### PR TITLE
demo doc updated

### PR DESCRIPTION
**Pull request by JrockiBedrock**
updated demo day doc to reflect current demo day process

### CODE REVIEW

File: src/docs/contribute/demo-day.md [hunk 1]
```diff
@@ -5,24 +5,27 @@ lang: en-US
...
 
 Make sure you check all these boxes before applying for demoing, or you will not get a demo spot. These spots go fast! 
 
 ### Pre-demo check list
 
-1. Is your dApp deployed & **live on Optimism**? Only dApp/features/etc that are ready to be use can be demo’d.
-1. Are you and **your team in the Optimism Discord**? The call is on Discord, so best get everyone on there! Make sure to tag the team member(s) who are presenting in the demo day channel.
-1. Do you have a **live or pre-recorded demo ready**? Your demo should be no longer than 7 minutes. Your demo can cover features, run throughs, promotions/special events etc. Just let us know!
-1. Will you be **available for a test run the day before** the demo at the same time as the demo? Nothing sucks more than being ready to demo but your sound isn’t working... we do a very quick check that all your settings are correct!
+1. Your dApp deployed & **live on Optimism**
```
Comment (Nicca42) [inline src/docs/contribute/demo-day.md:22]: Can we update this to live on OP Mainnet

**Review by Nicca42 — CHANGES_REQUESTED** [review]

File: src/docs/contribute/demo-day.md (hunk 1 above)
Comment (JrockiBedrock) [reply src/docs/contribute/demo-day.md]: updated language "live on Optimism" -->  "live on OP Mainnet"

**Review by Nicca42 — APPROVED** [review]

File: src/docs/contribute/demo-day.md [hunk 2]
```diff
@@ -5,24 +5,27 @@ lang: en-US
 
 ## About Demo Day
 
-Demo day was started by our amazing Ambassadors as a way for users to see all the cool dApps on Optimism! It runs every two weeks dApps have ~7 minutes to showcase their stuff. This can be anything from a full run through, to showing off a new feature, or promoting a special event on the dApp! 
+Demo day was started by the Optimism Ambassador Program as a way for users to see all the cool dApps on Optimism! It runs every Thursday at 4pm UTC.  dApps have ~15 minutes to showcase, we recommend 10 minutes to present and another 5 for questions. This can be anything from a full run through, to showing off a new feature, or promoting a special event on the dApp!
+
+
+To apply to be featured in our next Demo Day, head over to the [Demo-day channel](https://discord.com/channels/667044843901681675/960493820674465883) in our Discord (it's under events). 
+
+For further visibility projects are encouraged to add their project into our developers show and tell section here on [GitHub] https://github.com/ethereum-optimism/developers/discussions/categories/show-and-tell
```
Comment (Nicca42) [inline src/docs/contribute/demo-day.md:13]: This formatting is broken

**Review by Nicca42 — CHANGES_REQUESTED** [review]

File: src/docs/contribute/demo-day.md (hunk 2 above)
Comment (JrockiBedrock) [reply src/docs/contribute/demo-day.md]: just pushed the fix

**Review by Nicca42 — APPROVED** [review]

**Pull request merged**